wiremock/scripts/stubApis.ts

```diff
@@ -71,6 +71,24 @@ courses.forEach(course => {
   })
 })
 
+prisoners.forEach(async prisoner => {
+  stubs.push(async () =>
+    stubFor({
+      request: {
+        method: 'GET',
+        url: prisonerOffenderSearchPaths.prisoner.show({ prisonNumber: prisoner.prisonerNumber }),
+      },
+      response: {
+        status: 200,
+        headers: {
+          'Content-Type': 'application/json;charset=UTF-8',
+        },
+        jsonBody: prisoner,
+      },
+    }),
+  )
+})
```

Comment:
Could we not just run `stubPrisoners` after stubbing the rest in this file, instead of writing this in two places?